lib/plugins/wallet/tron/base.js

```diff
@@ -0,0 +1,190 @@
+const TronWeb = require('tronweb')
+const coins = require('@lamassu/coins')
+const { default: PQueue } = require('p-queue')
+
+const BN = require('../../../bn')
+
+let tronWeb = null
+
+const DEFAULT_PREFIX_PATH = "m/44'/195'/0'/0"
+const PAYMENT_PREFIX_PATH = "m/44'/195'/1'/0"
+
+const SWEEP_QUEUE = new PQueue({
+  concurrency: 3,
+  interval: 250,
+})
+
+function checkCryptoCode (cryptoCode) {
+  if (cryptoCode === 'TRX' || coins.utils.isTrc20Token(cryptoCode)) {
+    return Promise.resolve(cryptoCode)
+  }
+  return Promise.reject(new Error('cryptoCode must be TRX'))
+}
+
+function defaultWallet (account) {
+  const mnemonic = account.mnemonic
+  if (!mnemonic) throw new Error('No mnemonic seed!')
+
+  return TronWeb.fromMnemonic(mnemonic.replace(/[\r\n]/gm, ' ').trim(), `${DEFAULT_PREFIX_PATH}\/0`)
+}
+
+function paymentWallet (account, index) {
+  const mnemonic = account.mnemonic
+  if (!mnemonic) throw new Error('No mnemonic seed!')
+
+  return TronWeb.fromMnemonic(mnemonic.replace(/[\r\n]/gm, ' ').trim(), `${PAYMENT_PREFIX_PATH}\/${index}`)
```

Comment:
Should `\/` after `PAYMENT_PREFIX_PATH` be escaped?

Reply:
it's already an escaped forward slash